### ISSUE TITLE
Feature Request: Group By

### ISSUE THREAD

**Issue by jmwhite5** [thomasloven/lovelace-auto-entities#348]
I love this card.   One feature that would make it perfect for me is the ability to automatically group entities by zone.  I try to not to include redundant information in my entity names like type, zone, ... (e.g. I would never use "Kitchen Ceiling Light" as a name.  Just "Ceiling" because light and zones already capture that information.  So with this card, I sometimes end up with entities with similar names, but I don't know what zone they are in.

**Comment by robsonfelix** [comment]
Me too! Is there a way we can achieve this?

**Comment by albanleandri** [comment]
Yes, or to group entities by device. When filtering entities that haven't been seen for sufficient time, it tends to be the whole device that has an issue. This is one use case I can think about and that makes a group_by feature useful

**Comment by MartyTremblay** [comment]
I'd love to be able to group by area so that entities are displayed hierarchically under each respective area.

**Comment by sergiobaiao** [comment]
plus one for group by area

**Comment by dcapslock** [comment]
Check out [LFT fork](https://github.com/Lint-Free-Technology/lovelace-auto-entities) of auto-entities which has updated [sort options ](https://github.com/Lint-Free-Technology/lovelace-auto-entities#sorting-entities) as well as [muiltple sort levels](https://github.com/Lint-Free-Technology/lovelace-auto-entities#multiple-sort-levels) - which is pretty much group by, except you would expect a proper group by perhaps to have options to use a new heading if use in an entities card.

> method: Required One of domain, entity_id, name, friendly_name, device, area, state, attribute, last_changed, last_updated or last_triggered.
> - name — sorts by the entity's display name after any rename: transformation. Falls back to the Home Assistant friendly name when no rename is configured.
> - friendly_name — sorts by the entity's original Home Assistant friendly name, unaffected by any rename: configuration.
